server/index.js

```diff
@@ -3,13 +3,16 @@ const app = express();
 const port = 5000;
 const queries = require('./../db/queries');
 
+app.use(express.static('./public'))
+app.use(express.static('./client/dist'))
 
 app.use('/product/:id/description', (req, res) => {
     var { id } = req.params;
+    id = Number(id);
 
-    queries.getProductDesc()
+    queries.getProductDesc(id)
     .then(productDesc => {
-        res.end(productDesc);
+        res.end(JSON.stringify(productDesc[0]));
     })
```

Comment:
Does res.end send status codes? For clarify sake, it might be helpful to add a res.status(200).send(data) or res.status(500).send(error).

Reply:
oh yeah, thats the stuff i was talking about that i have to make a habit lol

and no, res.end doesn't send status codes